packages/odata-common/src/time.ts

```diff
@@ -17,8 +17,17 @@ export const durationRegexV4 =
  * Interface to represent Time or Duration.
  */
 export interface Time {
+  /**
+   * The amount of hours.
```

Comment:
[req] same for other properties.

```suggestion
   * The number of hours.
```